nvssclient/Controllers/RecordController.cs

```diff
@@ -251,36 +251,88 @@ public IActionResult AliasRecordHandler([FromBody] Dictionary<string, object> da
             return NoContent();
         }
 
+        // GET: List of Records
+        // Retrieves return all the unique business identifier sets and the status for the latest message that was sent with those business ids
+        [HttpGet]
+        public ActionResult<List<MessageItem>> GetAllRecordStatus()
+        {
+            try 
+            {            
+                // Return the most recent message with the given business identifiers and it's status
+                using (var scope = _scopeFactory.CreateScope()){
+
+                    // get a distinct list of business identifiers submitted to the API
+                    var context = scope.ServiceProvider.GetRequiredService<AppDbContext>();
+                    List<MessageItem> uniqueBussinessIds = context.MessageItems.OrderByDescending(s => s.CreatedDate).Select(p => new MessageItem()
+                    {
+                        DeathJurisdictionID = p.DeathJurisdictionID,
+                        CertificateNumber = p.CertificateNumber,
+                        DeathYear = p.DeathYear
+                    }).Distinct().ToList();
+                    if (uniqueBussinessIds == null) {
+                        Console.WriteLine("Error Retrieving status of death records");
+                        return NotFound("Records not found");
```

Comment:
Do we just want to return empty the empty set, so that this works the same even when empty?